docs/api/sandbox-option.md

```diff
@@ -1,9 +1,9 @@
 # `sandbox` Option
 
-> Create a browser window with renderer that can run inside chromium OS sandbox.
+> Create a browser window with a renderer that runs inside the Chromium OS sandbox.
```

Comment:
Passing `sandbox` alone does not enable chromium os sandbox, it simply creates a renderer that is compatible with it. In other words, a sandbox-compatible renderer will only communicate with system using IPC through main process.(read about `--enable-sandbox` option below)